network/validator/pubsub/topic_validator.go

```diff
@@ -103,15 +102,15 @@ func TopicValidator(log zerolog.Logger, codec network.Codec, peerFilter func(pee
 			return pubsub.ValidationReject
```

Comment:
eventually we'll want to add a slashing hook if `codec.Decode()` fails here, but it doesn't have to go into this PR. Can you either add it or add comment indicating that this is slashable and open an issue to track?

Reply:
Yes there is a ticket for it https://github.com/dapperlabs/flow-internal/issues/1762